js/libs/builder.js

```diff
@@ -650,7 +651,7 @@ new Vue({
 
       var $vm = this;
 
-      this.updateFormSettings(templateId, false);
+      templateId && this.updateFormSettings(templateId, false);
```

Comment:
Semantic coding principle: your code should say what it means.

```suggestion
      if (templateId) {
        this.updateFormSettings(templateId, false);
      }
```